flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -858,13 +862,15 @@ class MacOSFlutterLocalNotificationsPlugin
     bool badge = false,
     bool provisional = false,
     bool critical = false,
+    bool providesAppNotificationSettings = false,
   }) =>
-      _channel.invokeMethod<bool>('requestPermissions', <String, bool?>{
+      _channel.invokeMethod<bool?>('requestPermissions', <String, bool?>{
```

Comment:
```suggestion
      _channel.invokeMethod<bool>('requestPermissions', <String, bool?>{
```

In self-review, I see that this might not have been needed.